edx-platform/pearson-vue-theme/lms/templates/header/user_dropdown.html

```diff
@@ -13,6 +13,7 @@
 from openedx.core.djangoapps.user_api.accounts.toggles import should_redirect_to_order_history_microfrontend
 from openedx.core.djangoapps.user_api.accounts.utils import retrieve_last_sitewide_block_completed
 from openedx.features.enterprise_support.utils import get_enterprise_learner_generic_name, get_enterprise_learner_portal
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
```

Comment:
I think you are not using this import. @sergivalero20

Reply:
Indeed. I'll remove it.